static.md

```diff
@@ -20,3 +20,22 @@ module.exports = {
     }]
 };
 ```
+
+If you only want to copy _some_ of the files,
+you can select them using a `filter` predicate.
```

Comment:
I'm not sure our target audience is comfortable with the use of "predicate" here. @moonglum, do we have an established term for this?

Reply:
I would go with the term 'predicate'. We can also go with 'function' if you think that 'predicate' is not well established among our users. But I would just go with 'predicate' and wait if anyone has a problem with it.

Reply:
I believe this may be what they call “prioritising” or “wanting to ship the damn thing”.

How about an appeal to authority?
[The MDN docs on `Array.prototype.filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) avoid the term “predicate”.  Instead they say:
> The filter() method creates a new array with all elements that pass the test implemented by the provided function.